src/PriceCalculator.java

```diff
@@ -2,33 +2,28 @@
 
 public class PriceCalculator{
 
-    Sring subscription;
+    String subscription;
     double discount;
 
     public PriceCalculator(String subscription){
         if (subscription == "gold"){
-            this.discount = 0.15;
+            this.discount = 1-0.15;
         }
-        if (subscription == "platinum"){
-            this.discount = 0.1;
+        else if (subscription == "platinum"){
+            this.discount = 1-0.1;
         }
-        if (subscription == "silver"){
-            this.discount = 0.05;
+        else if (subscription == "silver"){
+            this.discount = 1-0.05;
+        }
+        else{
+            this.discount = 1.0;
         }
 
     }
 
 
-    public double calculatePrice(String subscription) {
```

Comment:
Sorry, my last comment was probably confusing. I just meant you could combine the two lines into one. We still need the cart item summation here with the full logic.

We should only be using this function in Order.java:
`orderPrice = priceCalc.calculatePrice(subscription)` rather than`orderPrice = calculatePrice(subscription)` because then we still have the code smell.